src/BuiltInTools/dotnet-watch/CommandLineOptions.cs

```diff
@@ -77,112 +75,131 @@ console output.
     public required IReadOnlyList<string> RemainingArguments { get; init; }
     public RunCommandLineOptions? RunOptions { get; init; }
 
-    public static CommandLineOptions? Parse(string[] args, IReporter reporter, out int errorCode, System.CommandLine.IConsole? console = null)
+    public static CommandLineOptions? Parse(string[] args, IReporter reporter, out int errorCode, TextWriter? output = null, TextWriter? error = null)
     {
-        var quietOption = new Option<bool>(new[] { "--quiet", "-q" }, "Suppresses all output except warnings and errors");
-        var verboseOption = new Option<bool>(new[] { "--verbose", "-v" }, "Show verbose output");
+        CliOption<bool> quietOption = new("--quiet", "-q")
```

Comment:
Can you please keep the previous style, i.e. `var _ = new CliOption<bool>(...)` instead of `CliOption<bool> _ = new(...)`? It'd be a) easier to read the PR and b) consistent with https://github.com/dotnet/sdk/pull/29131/files#diff-48c696e38dedfa782c9bb14f1e5749d861c4cd6ae825edda9497bb62deebdc3cR146

Reply:
@tmat I apology, I should not bring my `var` obsession from `dotnet/runtime` to this repo.

Changing that is possible, but it would take me quite a lot of time (I could automate it only partially) for the entire solution. Unless you are concerned about only this particular file?

Reply:
Just this particular file.

Reply:
@tmat done, PTAL